takecare19/takecare19
# Create a submission form

I created an admin resource submission form that can be reused on the public side (which is why I put in the effort to style it according to the mockups)

### Testing Instructions
- Go to `/admin`, click "New Resource'
- Enter resource info. Make sure form validation is working correctly
- Submit the resource
- Make sure the resource is displayed correctly on the resource list
